cuda_bindings/tests/test_nvjitlink.py

```diff
@@ -52,6 +52,19 @@ def ptx_header(version, arch):
 ]
 
 
+def check_nvjitlink_usable():
+    from cuda.bindings._internal import nvjitlink as inner_nvjitlink
+
+    if inner_nvjitlink._inspect_function_pointer("__nvJitLinkVersion") == 0:
+        return False
+    return True
+
+
+pytestmark = pytest.mark.skipif(
+    not check_nvjitlink_usable(), reason="nvJitLink not usable, maybe not installed or too old (<12.3)"
+)
```

Comment:
FYI @ksimpson-work the expanded CI caught this issue

Reply:
wonderful, good to see it doing its job